packages/documentation/src/stories/getting-started/migration-guide/migrationv6-7.component.ts

```diff
@@ -109,38 +109,135 @@ export class MigrationV56Component extends LitElement {
                   </p>
                 `
               : nothing}
-            <ul>
-              ${this.angular
-                ? html`
+
+            <section>
+              <h4>Styles</h4>
+
+              <ul>
+                <li class="mb-3">
+                  <p>
+                    <span class="tag tag-sm tag-danger">breaking</span> Removed the following
```

Comment:
Usually the tag is after the text, please reverse the order here or for the ngbAccordion whichever you prefer.